src/main/java/org/tema6/Calculator.java

```diff
@@ -0,0 +1,52 @@
+package org.tema6;
+
+import java.util.Scanner;
+
+
+
+public class Calculator {
+
+
+    static MetricConvertor metricConvertor = new MetricConvertor();
```

Comment:
```suggestion
    private final MetricConvertor metricConvertor = new MetricConvertor();
```
try to avoid making things static. In case we really need them static, try to make them final